control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1067,6 +1068,12 @@ func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedCont
 		return fmt.Errorf("failed to reconcile cloud controller manager: %w", err)
 	}
 
+	// Reconcile cloud credential operator
```

Comment:
let's put this under a `if hcp.platformtype == aws` and open up from there based on known need/use cases.